docs/COREDUMP_PACKAGE_METADATA.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Package Metadata for Core Files
+category: Interfaces
+layout: default
+---
+
+# Package Metadata for Core Files
+
+*Intended audience: hackers working on userspace subsystems that create ELF binaries
+or parse ELF core files.*
+
+## Motivation
+
+ELF binaries get stamped with a unique, build-time generated hex string identifier called
+`build-id`, which gets embedded as an ELF note called `.note.gnu.build-id`.
```

Comment:
Maybe add a link to https://fedoraproject.org/wiki/Releases/FeatureBuildId and some offical docs, if there are any?

Reply:
I don't know of any better link than that one